### PR TITLE
Update index.html - image added from unsplash.com

### PR DESCRIPTION
**Pull request by MichiBerger**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,6 +16,13 @@ <h1>Michi Berger</h1>
       <a href="#biography">Biography</a>
     </nav>
   </header>
+  
+  <figure>
+    <img src="https://source.unsplash.com/random/600x300" alt="">
+    <figcaption>A random picture from unsplash.com</figcaption>
+  </figure>
+  
+
```
Comment (PReichetanz) [inline index.html]: There should not be more than one empty line, so please delete it before merging 👯 
```suggestion
```

**Review by PReichetanz — COMMENTED** [review]
Nice work!

File: index.html (hunk 1 above)
Comment (MichiBerger) [reply index.html]: thx Paul...I´ve changed it now! I left just one empty line before and after `figure`

**Review by badabam — APPROVED** [review]
Well done!

**Pull request merged**